pkg/trace/config/config.go

```diff
@@ -53,7 +53,14 @@ type OTLP struct {
 	// automatically map Datadog Span Operation Names to an updated value. All entries should be key/value pairs.
 	SpanNameRemappings map[string]string `mapstructure:"span_name_remappings"`
 
-	// SpanNameAsResourceName uses the OTLP span name as the Datadog resource name.
+	// SpanNameAsResourceName specifies whether the OpenTelemetry span's name should be
+	// used as the Datadog span's operation name. By default (when this is false), the
+	// operation name is deduced from a combination between the instrumentation scope
+	// name and the span kind.
+	//
+	// For context, the OpenTelemetry Span Name's equivalent at Datadog is the resource name.
+	// The Datadog Span Name's equivalent in OpenTelemetry does not exist, but the span's kind
```

Comment:
Do you mean Datadog Span Operation Name here?

Reply:
I wanted to make it clear that this is about the [Name](https://github.com/DataDog/dd-trace-go/blob/main/ddtrace/tracer/span.go#L65) field, which is the "operation name". I wanted to emphasise that both at Datadog and in OpenTelemetry they are called "Span Names" but at Datadog it means "operation name". Do you still think it need's changing?

In other words, this is basically underlining the difference between the different span names. If I call one "span name" and the other one "span operation name" it sounds like two different things. They are both called "Name" within their structures and they are both the first argument to StartSpan in both APIs

Reply:
I think that datadog refers to "name" and "operation name" as the same thing is confusing since they sound like different things. So I think I would prefer to say Datadog span operation name here both times. 
Personally I would prefer we drop one of them entirely and only ever refer to it as operation name or 'name' and since 'operation name' is more specific I feel like that would win out)

Reply:
Sure. I think that makes sense. Can you please suggest how you'd like me to rephrase this? Is this what you were thinking?
```go
// The Datadog Span's Operation Name equivalent in OpenTelemetry does not exist, [...]
```

Reply:
I think that right there is perfect